Libraries/Azure.psm1

```diff
@@ -527,7 +527,7 @@ Function Invoke-AllResourceGroupDeployments($SetupTypeData, $CurrentTestData, $R
 			$terms = Get-AzMarketplaceTerms -Publisher $publisher -Product $offer -Name $sku -ErrorAction SilentlyContinue
 			if ($terms -and !$terms.Accepted) {
 				Write-LogInfo "Accept terms for Publisher $publisher, Product $offer, Name $sku"
-				$terms | Set-AzMarketplaceTerms -Accept | Out-Null
+				Set-AzMarketplaceTerms -Name $sku -Product $offer -Publisher $publisher -Accept
```

Comment:
why this `$terms | Set-AzMarketplaceTerms -Accept | Out-Null` doesn't work

Reply:
Seems like the upgrade in AZ ps changed the syntax. 
Error log from v2 execution:
11/18/2022 13:36:23 : [INFO ] Accept terms for Publisher Canonical, Product 0001-com-ubuntu-server-focal, Name 20_04-lts-gen2
11/18/2022 13:36:23 : [ERROR] Exception detected. Source : DeployVMs()
11/18/2022 13:36:23 : [ERROR] EXCEPTION : Cannot bind argument to parameter 'Name' because it is an empty string.
11/18/2022 13:36:23 : [ERROR] Source : Line 530 in script .\Libraries\Azure.psm1.
11/18/2022 13:36:23 : [ERROR] EXCEPTION: VMData is empty (null). Aborting the testing.
11/18/2022 13:36:23 : [ERROR] Calling function - .
11/18/2022 13:36:23 : [ERROR] Source: Line 805 in script .\TestControllers\TestController.psm1.